db/migrate/20150123014304_create_recipe_ingredient_lists.rb

```diff
@@ -0,0 +1,17 @@
+class CreateRecipeIngredientLists < ActiveRecord::Migration
+  def change
+    create_table :recipe_ingredient_lists do |t|
+      t.integer :recipe_id
+      t.integer :ingredient_id
+      t.integer :recipe_amount_us
```

Comment:
The names are too complicated! Suggestion: quantity_us, unit_us, quantity_metric, unit_metric (no need to repeat recipe_ !)